cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -57,6 +57,23 @@ define(
                 jasmine.Ajax.uninstall();
             });
 
+            it('renders correct text in file drag/drop area', function() {
+                var messages = {
+                        '.video-upload-text': 'Drag and drop or click here to upload video files.',
```

Comment:
Drag and drop your video files here, or click this space to browse for files on your computer.